tritium-tracing/src/main/java/com/palantir/tritium/tracing/RemotingCompatibleTracingInvocationEventHandler.java

```diff
@@ -118,43 +151,18 @@ private static boolean isUsingMultipleTracers(Class<?> tracersClass)
                 String expectedTracingPackage = com.palantir.tracing.Tracers.class.getPackage().getName();
                 String actualTracingPackage = wrappedTrace.getClass().getPackage().getName();
                 if (!Objects.equals(expectedTracingPackage, actualTracingPackage)) {
-                    logger.error("Multiple tracing implementations detected, expected '{}' but found '{}',"
-                                    + " using legacy remoting3 tracing for backward compatibility",
-                            SafeArg.of("expectedPackage", expectedTracingPackage),
-                            SafeArg.of("actualPackage", actualTracingPackage));
+                    if (shouldLogFallbackError.compareAndSet(false, true)) {
+                        logger.error("Multiple tracing implementations detected, expected '{}' but found '{}',"
+                                        + " using legacy remoting3 tracing for backward compatibility",
+                                SafeArg.of("expectedPackage", expectedTracingPackage),
+                                SafeArg.of("actualPackage", actualTracingPackage),
+                                new SafeIllegalStateException("Multiple tracing implementations detected"));
```

Comment:
I'm not sure the exception stack trace will help us debug, maybe the ClassLoader.toString?